docker/phpfpm/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-fpm
+FROM php:8.1.0-fpm
```

Comment:
Why not the latest fix, though?
```suggestion
FROM php:8.1.13-fpm
```

Reply:
just because my IDE did not find it as stable , let me take a look and commit that

Reply:
found this : 
https://hub.docker.com/layers/library/php/8.1.13-fpm/images/sha256-febc9b5002105e4ff64de25d30d180f12eca2dc438b0dc1559d8561dcfda1fbc?context=explore
so it looks ok by me